.github/workflows/php-static.yml

```diff
@@ -0,0 +1,96 @@
+name: Static analysis
+
+on:
+ workflow_call:
+  inputs:
+   php_versions:
+    required: true
+    type: string
+
+jobs:
+  phpstan:
+    strategy:
+      matrix:
+        php: ${{ fromJSON(inputs.php_versions) }}
+        stability: [prefer-lowest, prefer-stable]
+
+    name: PHPStan - ${{ matrix.php }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          extensions: mbstring, intl
```

Comment:
Do we need to accept this as an definable input with a sane default? I know we use other extensions in VOL for instance.

Reply:
As discussed the other day I think for test it is useful